src/components/Theme/constants.css

```diff
@@ -59,21 +59,21 @@
 @color-primary-500: #353FF4;
 @color-icon-temporary: #EDEEEF;
 @color-neutral-600: #A2A8B3;
-@color-primary-700: #636670;
 
 @color-background: @color-white;
 @color-border: @color-neutral-600;
 @color-divider: #DEE1E4;
 @color-spinner: @color-neutral-600;
-@color-body-text: @color-primary-700;
+@color-body-text: #1E1E24;
+@color-title-text: #2B2D33;
+@color-subtitle-text: @color-body-text;
 
-@color-text-dark: #2C3E4F;
+@color-text-dark: #1E1E24;
```

Comment:
`@color-body-text` and `@color-text-dark` have the same value. When do we use `@color-text-dark`, and why can we not use `@color-body-text directly?

Reply:
Good point. `@color-text-dark` seems to be used across a few components' stylesheets (`CrossDeviceLink`, `PhoneNumberInput`, `NavigationBar`, `PoAIntro` to name a few) but they really are the same thing now. Will update.